### PR TITLE
Request for code review 

### PR DESCRIPTION
**Pull request by TSHEPO-CLOUD**
# Cloning [Mint.com](http://www.mint.com/)  signup page.

In this project, we:
- Added and displayed HTML Form to accept user input on the page
- Embedded Images by using sprites and icons
- Made section of the page responsive
- Made use of CSS3 to style the Page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  padding: 0.1em;
+}
+
+header nav button {
+  background-color: #0077c5;
+  display: inline-block;
+  border: 1px solid #222;
+  margin-right: 3em;
+  font-size: 14px;
+  color: #fff;
```
Comment (bertil291utn) [inline index.css:37]: ## Using the Same Rule Repetitively[1]
- [ ] Avoid using the same rule many times. Try this

```
.color-white{
  color:#fff;
}
```

In the HMTL file in every element that uses white color

`<div class="color-white"></div>`

- Please once you create the class, delete these rules in the entire CSS file


###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: index.css [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  font-size: 0.9em;
+}
+
+/* == header styles == */
+header {
+  padding: 0.4em;
+}
+
+nav {
+  display: flex;
```
Comment (bertil291utn) [inline index.css:20]: ## Using the Same Rule Repetitively[1]
- [ ] Avoid using the same rule many times. Try this

```
.flex{
  display:flex;
}
```

In the HMTL file in every element that uses white color
`<div class="flex"></div>`

- Please once you create the class, delete these rules in the entire CSS file


###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: index.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                        <label for="pwd">Password</label>
+                        <input type="password" name="pwd" id="pwd">
+                    </div>
+                    <div class="form-control hide">
+                        <label for="pwd2">Confirm password</label>
+                        <input type="password" name="pwd2" id="pwd2">
+                    </div>
+                    <div class="form-control">
+                        <button type="submit">
+                            <i class="material-icons icons" style="font-size: 14px;">lock</i>
```
Comment (bertil291utn) [inline index.html:68]: ## Using Inline Styles[1]

- Avoid using inline styles. 

- [ ] Please move to your CSS file all your `style=""` properties



###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Comment by belsman** [comment]
> **Status:** Changes requested 🔴 :shipit:? Not yet.
> 
> # Hey guys, you've been doing a great job, and also I hope you enjoyed coding.
> ### Good job on your changes, however, some requests are not changed.
> * [ ]  It shows the Mint logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
> * [ ]  It shows the Quickbooks logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
> * [ ]  It shows the Turbotax logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
> 
> Should be text but you have an image
> 
> image
> 
> * [ ]  **Learn more** should be in the same line as [the original](https://accounts.intuit.com/signup.html)
> 
> image
> 
> * [ ]  Don't forget also work in the inline comments requirements below this.
> 
> Guys work in the requested changes above and request another review. Good luck.
> > “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: index.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        </nav>
+    </header>
+    <main>
+        <aside class="col-1"></aside>
+        <section class="container">
+            <div class="container-header">
+                <div class="banner"></div>
+                <ul class="menu-sprite">
+                    <li class="icon logo-1"></li>
+                    <li class="sprint mint"></li>
```
Comment (expjazz) [inline index.html:35]: This should not be a background image, but a text with mint as a content. The same goes for the Quickbooks and Turbotax below (lines 37 and 39)

**Pull request merged**